src/components/Sidebar/SidebarItem.tsx

```diff
@@ -137,45 +65,34 @@ const SidebarItemArrowIcon = styled.span`
   }
 `;
 
-const SidebarItem: FC = memo(() => {
+const SidebarItem: FC<MenuItemProp> = memo(({menuItem, level = 0}) => {
+  const {icon, title, children} = menuItem;
   const [isOpened, setIsOpened] = useState(false);
-  const settingsToogler = useCallback(() => {
-    setIsOpened(!isOpened);
-  }, [isOpened]);
+  const toggleOpen = useCallback(() => {
+    if (children) {
+      setIsOpened(!isOpened);
+    }
+  }, [isOpened, children]);
 
   return (
-    <>
-      {MenuItems.map(({icon, title, children}) => (
-        <ItemsWrapper key={title}>
-          <SidebarItemWrapper onClick={children && settingsToogler}>
-            <SidebarItemIcon>{icon}</SidebarItemIcon>
-            <SidebarItemName>{title}</SidebarItemName>
-            {children && (
-              <SidebarItemArrowIcon>
-                {isOpened ? (
-                  <strong>
-                    <Arrow />
-                  </strong>
-                ) : (
-                  <Arrow />
-                )}
-              </SidebarItemArrowIcon>
+    <ItemsWrapper paddingLeft={`${48 * level}px`}>
```

Comment:
It's best to put 48 in a const outside of the component. Because "what does 48 mean? Where is it from? Why 48?".
But if you have const basePadding = 48, you instantly know what it is and what it's for.

Imagine if you have many random numbers in your code, like 10 for example. You will get lost and forget what each number is for.